docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   postgres-botner:
     container_name: postgres-botner
-    image: postgres:12
+    image: postgres
```

Comment:
Why this change?

Reply:
Was trying something. Do you want me to revert it back? It downloads v14 this way...